### ISSUE TITLE
[ A11y | shellbar | button count ]: the text only on button for tooltip needs count if applicable

### ISSUE THREAD

**Issue by nopearson** [SAP/ui5-webcomponents#11198]
### Bug Description

The shellbar item count should be read by screenreader in tooltip or accessibleNameRef property - only have text now

### Affected Component

shellbar

### Expected Behaviour

screenreader should be able to discern the count:

![Image](https://github.com/user-attachments/assets/ba139790-30d6-4840-8755-de24ecc761f2)

### Steps to Reproduce

1.
2.
3.
...

### Priority

High

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Organization

sap fieldglass

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Reproducible, tested with JAWS 2024.2409.2 on [this](https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cbjxoZWFkPlxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbiAgICA8bGluayByZWw9XCJzdHlsZXNoZWV0XCIgaHJlZj1cIi4vbWFpbi5jc3NcIj5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpO2hlaWdodDogNDAwcHg7XCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1zaGVsbGJhciBpZD1cInNoZWxsYmFyXCIgcHJpbWFyeS10aXRsZT1cIkNvcnBvcmF0ZSBQb3J0YWxcIiBzZWNvbmRhcnktdGl0bGU9XCJzZWNvbmRhcnkgdGl0bGVcIlxuICAgICAgICBub3RpZmljYXRpb25zLWNvdW50PVwiOTkrXCIgc2hvdy1ub3RpZmljYXRpb25zPVwiXCI-XG5cdFx0PHVpNS10b2dnbGUtYnV0dG9uIGljb249XCJzYXAtaWNvbjovL2RhXCIgc2xvdD1cImFzc2lzdGFudFwiPjwvdWk1LXRvZ2dsZS1idXR0b24-XG4gICAgICAgIDxpbWcgc2xvdD1cImxvZ29cIiBzcmM9XCJodHRwczovL3NhcC5naXRodWIuaW8vdWk1LXdlYmNvbXBvbmVudHMvaW1hZ2VzL3NhcC1sb2dvLXN2Zy5zdmdcIiAvPlxuICAgICAgICA8dWk1LXNoZWxsYmFyLWl0ZW0gaWQ9XCJjdXN0b21JdGVtXCIgaWNvbj1cImFjdGl2YXRlXCIgY291bnQ9XCI1MFwiIHRleHQ9XCJTaGVsbEJhciBJdGVtXCI-PC91aTUtc2hlbGxiYXItaXRlbT5cbiAgICAgICAgPHVpNS1idXR0b24gaWNvbj1cIm5hdi1iYWNrXCIgc2xvdD1cInN0YXJ0QnV0dG9uXCI-PC91aTUtYnV0dG9uPlxuXG4gICAgICAgIDx1aTUtaW5wdXQgc2xvdD1cInNlYXJjaEZpZWxkXCI-PC91aTUtaW5wdXQ-XG5cbiAgICAgICAgPHVpNS1saSBzbG90PVwibWVudUl0ZW1zXCI-QXBwbGljYXRpb24gMTwvdWk1LWxpPlxuICAgICAgICA8dWk1LWxpIHNsb3Q9XCJtZW51SXRlbXNcIj5BcHBsaWNhdGlvbiAyPC91aTUtbGk-XG4gICAgICAgIDx1aTUtbGkgc2xvdD1cIm1lbnVJdGVtc1wiPkFwcGxpY2F0aW9uIDU8L3VpNS1saT5cbiAgICA8L3VpNS1zaGVsbGJhcj5cbiAgICA8dWk1LXBvcG92ZXIgaWQ9XCJhY3Rpb24tcG9wb3ZlclwiIHBsYWNlbWVudD1cIkJvdHRvbVwiPlxuICAgICAgICA8ZGl2IGNsYXNzPVwiYWN0aW9uLXBvcG92ZXItaGVhZGVyXCI-XG4gICAgICAgICAgICA8dWk1LXRpdGxlIHN0eWxlPVwicGFkZGluZzogMC4yNXJlbSAxcmVtIDByZW0gMXJlbVwiPkFuIEtpbXVyYTwvdWk1LXRpdGxlPlxuICAgICAgICA8L2Rpdj5cbiAgICAgICAgPGRpdiBjbGFzcz1cImFjdGlvbi1wb3BvdmVyLWNvbnRlbnRcIiBzdHlsZT1cIm1hcmdpbi10b3A6IDFyZW07XCI-XG4gICAgICAgICAgICA8dWk1LWxpc3Qgc2VwYXJhdG9ycz1cIk5vbmVcIj5cbiAgICAgICAgICAgICAgICA8dWk1LWxpIGljb249XCJzeXMtZmluZFwiPkFwcCBGaW5kZXI8L3VpNS1saT5cbiAgICAgICAgICAgICAgICA8dWk1LWxpIGljb249XCJzZXR0aW5nc1wiPlNldHRpbmdzPC91aTUtbGk-XG4gICAgICAgICAgICAgICAgPHVpNS1saSBpY29uPVwiZWRpdFwiPkVkaXQgSG9tZSBQYWdlPC91aTUtbGk-XG4gICAgICAgICAgICAgICAgPHVpNS1saSBpY29uPVwic3lzLWhlbHBcIj5IZWxwPC91aTUtbGk-XG4gICAgICAgICAgICAgICAgPHVpNS1saSBpY29uPVwibG9nXCI-U2lnbiBvdXQ8L3VpNS1saT5cbiAgICAgICAgICAgIDwvdWk1LWxpc3Q-XG4gICAgICAgIDwvZGl2PlxuICAgIDwvdWk1LXBvcG92ZXI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9BdmF0YXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0J1dHRvbi5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvSW5wdXQuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0xpc3RJdGVtU3RhbmRhcmQuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L1BvcG92ZXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L1RpdGxlLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9Ub2dnbGVCdXR0b24uanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0xpc3QuanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWZpb3JpL2Rpc3QvU2hlbGxCYXIuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1maW9yaS9kaXN0L1NoZWxsQmFySXRlbS5qc1wiO1xuXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9kYS5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvZGEtMi5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvbmF2LWJhY2suanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L3N5cy1maW5kLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9zZXR0aW5ncy5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvZWRpdC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3Qvc3lzLWhlbHAuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy1pY29ucy9kaXN0L2xvZy5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvYWN0aXZhdGUuanNcIjtcblxuY29uc3Qgc2hlbGxiYXIgPSBkb2N1bWVudC5nZXRFbGVtZW50QnlJZChcInNoZWxsYmFyXCIpO1xuY29uc3QgYWN0aW9uUG9wb3ZlciA9IGRvY3VtZW50LmdldEVsZW1lbnRCeUlkKFwiYWN0aW9uLXBvcG92ZXJcIik7XG5cbnNoZWxsYmFyLmFkZEV2ZW50TGlzdGVuZXIoXCJ1aTUtcHJvZmlsZS1jbGlja1wiLCAoZXZlbnQpID0-IHtcbiAgICBhY3Rpb25Qb3BvdmVyLm9wZW5lciA9IGV2ZW50LmRldGFpbC50YXJnZXRSZWY7XG5cdGFjdGlvblBvcG92ZXIub3BlbiA9IHRydWU7XG59KTtcblxuWy4uLmRvY3VtZW50LnF1ZXJ5U2VsZWN0b3JBbGwoXCJ1aTUtdG9nZ2xlLWJ1dHRvblwiKV0uZm9yRWFjaChlbCA9PiB7XG5cdGVsLmFkZEV2ZW50TGlzdGVuZXIoXCJjbGlja1wiLCBldmVudCA9PiB7XG5cdFx0Y29uc3QgdG9nZ2xlQnV0dG9uID0gZXZlbnQudGFyZ2V0O1xuXHRcdHRvZ2dsZUJ1dHRvbi5pY29uID0gdG9nZ2xlQnV0dG9uLnByZXNzZWQgPyBcInNhcC1pY29uOi8vZGEtMlwiIDogXCJzYXAtaWNvbjovL2RhXCI7XG5cdH0pO1xufSk7In19) sample - the ui5-shellbar-item with id="customItem".

The count is indeed not read, as I can see form the ShellBar Item the tooltip set results in the string for the text and is not linked to count.

Could you please take a look?

Best,
Siyana
